app/views/topology/service.js

```diff
@@ -1346,6 +1349,29 @@ YUI.add('juju-topology-service', function(Y) {
     },
 
     /**
+      Returns a service name which is a maximum of 10 characters wide.
+
+      @method truncateServiceName
+      @param {Object} service The service model
+      @return {String} The truncated service display name.
+    */
+    truncateServiceName: function(service) {
+      var name = service.displayName;
+      if (service.pending) {
```

Comment:
does this even need to be put only when .pending? I mean if we're going to use () as a trigger for things should we just not allow names that have that from users?